Gemfile

```diff
@@ -118,6 +118,7 @@ group :development do
 end
 
 group :test do
+  gem "capybara"
```

Comment:
Why do we need capybara?

Reply:
Echo'ing this - seems like a lot for just this content matcher?  https://github.com/DFE-Digital/early-careers-framework/pull/318/files#diff-27bfa81d2f2f9f521f25ada1cab4cd1d875238941cabbbf6b91127de926883a9R37

Reply:
@jamesrose Agree - removed it for now. However I'd love to add this at later stage so that we can test our component content in isolation.